pkg/cmd/service/service_test.go

```diff
@@ -93,6 +99,11 @@ func ensureServiceUp(t *testing.T, transportCredentials credentials.TransportCre
 		return nil
 	}, policy)
 	require.NoError(t, err)
+
+	if useHTTPToo {
```

Comment:
Can we rename this to `httpHealthCheck`?

Reply:
Done. Thanks!